### PR TITLE
Add org.kde.krusader

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)* Same as Dolphin
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/utilities/krusader/-/merge_requests/85
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/utilities/krusader/-/merge_requests/85

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.krusader

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by Mailaender** [comment]
bot, build org.kde.krusader

**Comment by travier** [comment]
Manifest LGTM. Not yet tested. Can you add Flatpak external data checker config too? Thanks!

**Comment by travier** [comment]
https://release-monitoring.org/project/9651/

**Comment by Mailaender** [comment]
It is already integrated.

**Comment by travier** [comment]
Whoops, sorry.

**Comment by barthalion** [comment]
Needs approval from KDE team.

File: org.kde.krusader.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "command": "krusader",
+  "rename-icon": "krusader_user",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--filesystem=home",
+    "--filesystem=/run/media",
+    "--filesystem=/media"
```
Comment (travier) [inline org.kde.krusader.json:15]: Missing `--device=dri`

**Comment by travier** [comment]
Screenshot_20220523_085422

We should probably include/duplicate some of those apps in this one to make it really useful.

**Comment by travier** [comment]
Screenshot_20220523_085542

**Comment by travier** [comment]
Appears to work well apart from that.

**Comment by travier** [comment]
So I'd say we merge it and add more later? WDYT @tsdgeos ?

**Comment by travier** [comment]
(still need the dri change)

**Comment by Mailaender** [comment]
Added some more extract utilities.

**Comment by Mailaender** [comment]
bot, build org.kde.krusader

**Comment by tsdgeos** [comment]
22:45:45.037-warning kf.solid.backends.udisks2 Solid::Backends::UDisks2::Manager::introspect@160 # Failed enumerating UDisks2 objects: "org.freedesktop.DBus.Error.Disconnected" 
 "Not connected to D-Bus server"

Seems like this needs some dbus access to udisks

**Comment by tsdgeos** [comment]
Also as with the other app, this application has system tray support so needs the system tray dbus talk line

**Comment by Mailaender** [comment]
Added

bot, build org.kde.krusader

**Comment by tsdgeos** [comment]
There is what i think is a big issue and is that you can't open files from krusader.

if i run dolphin from flatpak and open the spreadsheet that i have in my home folder, it'll open libreoffice from my system to handle it

but if  i run krusader from flatpak and open the spreadsheet that i have in my home folder, it'll say "I don't know what do do with this" and offer me sad dialog with virtually no available apps available to open the file

Without fixing this i fell like at most we could put this in beta channel

**Comment by travier** [comment]
Indeed, had not seen that. Definitely beta only.

**Comment by barthalion** [comment]
I don't think it makes sense to put something somewhat broken in beta either.

**Comment by barthalion** [comment]
…but yeah, it's also not a hill I'm willing to die on.

/merge:beta

**Comment by travier** [comment]
> I don't think it makes sense to put something somewhat broken in beta either.

Agree it's not great. But I think it's easier for folks to contribute if the app is already there in beta.

**Comment by Mailaender** [comment]
We had to try to find out it does not (yet) work.

**Pull request closed without merging**